[W10-3] EventOrganiser

kengjichow opened the pull request.

AceCentury commented on src/main/java/seedu/address/logic/commands/AddEventCommand.java:
```diff
@@ -0,0 +1,50 @@
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD;
+    public static final String MESSAGE_SUCCESS = "New event added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This event already exists in the address book";
```
Perhaps this variable should be named MESSAGE_DUPLICATE_EVENT ?

AceCentury commented on src/main/java/seedu/address/model/event/UniqueEventList.java:
```diff
@@ -0,0 +1,133 @@
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * A list of persons that enforces uniqueness between its elements and does not allow nulls.
```
Similarly, your javadoc comments should be updated to reflect that the function is for "Event"

AceCentury commented on src/test/java/seedu/address/logic/commands/AddCommandTest.java:
```diff
@@ -152,6 +163,11 @@ public void redoAddressBook() {
         }
+
+        @Override
+        public void addEvent(Event toAdd) {
```
You can consider creating a separate test class, e.g. AddEventCommandTest, to test the AddEventCommand class which you created. 

This would be more ideal as it would allow you to isolate issues quicker. E.g. if the AddCommandTest fails, you would need to identify whether it is the AddCommand **or** the AddEventCommand that is the one failing.

AceCentury reviewed (commented):
Good progress so far, especially with the extensive code modifications!

Here are some things to take note:

* Try to replace the build badges (Travis, Appveyor etc) to with that of your team repo instead of AB4 repo.

* Your homepage (https://cs2103-ay1819s1-w10-3.github.io/main/) and netlify preview don't seem to be synchronised with your master branch. Check that Travis is set up correctly to deploy pages.

* Try to add a UI mockup to replace the existing one if possible.

AceCentury commented on docs/DeveloperGuide.adoc:
```diff
@@ -831,29 +875,282 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+1.	Event organiser searches for other users.
+2.	System shows user all the other users.
+3.	User chooses a user that he/she wants to invite
```
Try to have consistent punctuation. Some of your sentences end with full stops, some don't. I would suggest adding full stops all of sentences in all your use cases.

AceCentury commented:
@yao-feng Don't forget to update your profile picture.

yaofeng-wang commented:
Thanks for the reminder :)

On Wed, Sep 26, 2018 at 10:34 PM Ryan Teo <notifications@github.com> wrote:

> @yao-feng <https://github.com/yao-feng> Don't forget to update your
> profile picture.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nus-cs2103-AY1819S1/addressbook-level4/pull/5#issuecomment-424738239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASu0ZcWSUd9plW45FT6EOsWvatgbkOlcks5ue5BlgaJpZM4WqPPG>
> .
>

AceCentury commented:
I've noticed that you all had a number of commits to "Fix Style". 

I would advise you to run the tests locally first, fix any relevant errors **then** make the commit.

adjscent commented:
> I've noticed that you all had a number of commits to "Fix Style".
> 
> I would advise you to run the tests locally first, fix any relevant errors **then** make the commit.

The issues do not appear in the local tests, and appear only on Travis CI.

AceCentury commented:
Good job on your progress towards v1.1!

Here are some things to take note/update:

- Travis build status badge should point to the build status of your own repo, not se-edu/AB-4. Other badges should be updated to match your project or removed.

- The navigation bar at the top of your website preview should be tweaked/removed to reflect the homepage of your product.
![image](https://user-images.githubusercontent.com/30991447/46582085-5f60da00-ca74-11e8-8c77-0354659bf21d.png)

- About Us page - @theJrLinguist your photo doesn't seem to be rendering properly on the netlify preview. @yao-feng if possible, try to update your photo such that your face is clearly visible. I find that your face is a little dim in the current photo.

- Developer Guide

All sentences in use cases should have full stops.
![image](https://user-images.githubusercontent.com/30991447/46582157-73590b80-ca75-11e8-9a59-e9dc0ed75c88.png)

Non-functional requirements could be expanded a little bit.

Division of work should also have a consistent format. Some users don't have any designated "feature" (?) to work on.

![image](https://user-images.githubusercontent.com/30991447/46582180-b915d400-ca75-11e8-8d03-1ecd648b2f85.png)

AceCentury commented:
Minor typo for the **Extensions** of U07

![image](https://user-images.githubusercontent.com/30991447/46650708-2dfd2100-cbd0-11e8-93f8-60a9ac8bbf72.png)

AceCentury commented:
### v1.2 Feedback
The comments below are related to your Developer Guide

**Section 2: Design**
- I can see that some of the diagrams have been updated, but do remember to update all the outdated diagrams (e.g. UI)

**Section 3: Implementation**
- Would be good if you can update the Undo/Redo command references from AddressBook to EventOrganiser.

- For a sequence diagram in the **Poll Feature**, it would be good if the arrow could point to the middle of the object.

![image](https://user-images.githubusercontent.com/30991447/47268758-6b29c180-d587-11e8-987d-4dd0500c9867.png)

- For the sequence diagram in the **TimeTable Feature**, it would be good if the self-invocations have a return arrow.

![image](https://user-images.githubusercontent.com/30991447/47268786-b9d75b80-d587-11e8-90fc-1bc72aa892ba.png)

- That same sequence diagram also happens to have an instance of AddressBookParser. I am not sure if it's intentional or it was overlooked, but just pointing it out. 

Overall, I can see that effort has been put in to describe the Implementations of specific components/features. It would be good if each component/feature could have their own class and sequence diagrams. 

So far, the **Poll Feature** is very well documented in terms of implementation and design considerations. Good job!

**Contact Us Page**

- Please update the details on the **Contact Us** page and link to your issue tracker.

![image](https://user-images.githubusercontent.com/30991447/47268832-4e41be00-d588-11e8-95a5-b5592065a8e3.png)

AceCentury commented:
**Following comments are in relation to Developer Guide**

**Chapter 2: Design**

- I did not observe any changes in this section except for **Model and Storage**. Correct me if I may have missed anything.
- For the diagram in **Model Component**, the multiplicity is somewhat unclear. Are you implying that both have a multiplicity of 1? Ideally, try to be more explicit and put "1" at both ends if that is the case.

![image](https://user-images.githubusercontent.com/30991447/47961547-205c7f00-e048-11e8-9765-ab08f6e52c34.png)


**3.1. Login/logout feature**

- The sequence diagram is very small. Yes, I can always zoom in, but not sure if you might be penalised for that.

- For self-invocations in your sequence diagram, I am unsure if showing it as "stacked bars" instead of "half overlapping" bars is accepted. I understand why you might do this - showing overlaps with the same colour is impossible. A possible workaround could be to change the colour of the self-invocation bars, which was what I did. I will check with the teaching team and get back to you guys on this.

- Overall very well documented with UI screenshots. You might also want to include a class diagram.

**3.2. Poll feature**

- The first sequence diagram has a return arrow which is extended too long.

![image](https://user-images.githubusercontent.com/30991447/47961651-e42a1e00-e049-11e8-94af-d4e10f467833.png)

- There seems to be a typo in your alternatives "Alternative 1::". There are repeated colons. Also, it would be good to indicate your current choice of implementation e.g. Alternative 1(Current Choice).

- Overall well documented.  It would be good to include UI screenshots and/or a class diagram.

**3.3. Schedule feature**

- There seems to be a typo in your alternatives "Alternative 1::". There are repeated colons. Also, it would be good to indicate your current choice of implementation e.g. Alternative 1(Current Choice).

- Very detailed analysis on time and space complexity.

- Having some diagrams (class, sequence, or perhaps the "Architecture" of your schedule feature) might be useful to aid your explanations.

**3.4. TimeTable feature**

- Minor typos in your explanation: "All of these must be **fulfiled** or else a Invalid Nusmods link will be **returend** to the application."

- Might want to consider adding return arrows on self-invocation in sequence diagram.

- Would be good to have alternative implementations and evaluate their Pros and Cons.

**3.5. Add/Delete Friend feature**

- The text in the sequence diagram is really small in my opinion. As mentioned earlier, I can always zoom in, but not sure if you'll be penalised. 

- Try to render the diagram without all the red "spellcheck" underlines. 

- Might be good to add a bigger gap between Person p1 and p2 so that you can draw the lifeline. At this point I can't really tell whether they continue to live on or a destroyed. 

- **IMPORTANT:** In your add friend command, do you **create** 2 new Persons or do you **use existing** Persons? Your diagram seems to imply that you are creating 2 new Persons. 

- Having a class diagram might be helpful. Your guide does mention "we represents a friend of a Person using a Friend class." and it would be easier to see how it is "linked" to other classes.

- Overall well documented, but it would be good if you listed alternative implementations and their Pros and Cons.

**Appendix H: Instructions for Manual Testing**

- Don't forget to update this section.

**Overall Comments**

In general, your developer guide is on track. Minor amendments and inclusions can be added to make it better. Also, take note of formatting and ensure that it is consistent. For example, in Section 3, "Login/logout feature" does not bold the Alternatives, but other features do.

AceCentury commented:
**Important**

Stacked activation bars for self-invocation **are wrong**. Example:
![image](https://user-images.githubusercontent.com/30991447/47964606-89a7b680-e077-11e8-922b-f418a84723ee.png)

I would recommend using different colours for self-invocation activation bars so that you may show the overlap accordingly. An example is in the website/textbook:

![image](https://user-images.githubusercontent.com/30991447/47964626-b78cfb00-e077-11e8-8b67-7ce8d02d8ed1.png)


I have also previously commented that some of your diagrams are very small, especially if the interactions spans across components. For those diagrams, please refer to this clarification I received from Professor

"Reference frames are an optional topic -- peer reviewers might not understand them. The better thing to do is not to show low-level interactions across components. instead, they should do multi-level diagrams. High-level diagram at component level, and further diagrams to show what happens inside each component, if necessary."

In short, while reference frames _can_ be used to solve the issue of having too much information in a diagram, it might cause some confusion for peer reviewers. You are advised to do multi-level diagrams. Take note of this, especially for your sequence diagrams.

yaofeng-wang commented:
@AceCentury Thank you for the review! Will update them ASAP!

yaofeng-wang commented:
@AceCentury 
For login/logout feature:
1. Updated both sequence diagrams.
2. Added manual tests.
3. Corrected inconsistencies in format for "Alternatives" part.

Did not include a class diagram because for this feature, I've merely add in additional methods/attributes into classes already existing in AddressBook Level 4 e.g. modelManager, Person, etc. If I were to include a class diagram, most of the methods/attributes would have existed in AddressBook Level 4. Would you still recommend including a class diagram?

AceCentury commented:
Developer Guide review performed on Thursday (1 day after tutorial) as requested by team.

**Disclaimer**: This final review is meant to be brief check to spot any major errors or potential issues which need to be fixed. It will not be as comprehensive as my previous reviews where I point out other minor errors (such as formatting etc.) The onus is on you guys to thoroughly proofread and make the necessary amendments, if any.

**Poll Feature**

- It would be good if the text in the sequence diagram could be made bigger.

**TimeTable Feature**

- The sequence diagram is very small. Try to make it bigger (including the text). 

**Add/Delete Friend Feature**

- The text used for function calls and return values is extremely small. Try to make the text larger. I can't read it without zooming in.

AceCentury commented:
> @AceCentury
> For login/logout feature:
> 
> 1. Updated both sequence diagrams.
> 2. Added manual tests.
> 3. Corrected inconsistencies in format for "Alternatives" part.
> 
> Did not include a class diagram because for this feature, I've merely add in additional methods/attributes into classes already existing in AddressBook Level 4 e.g. modelManager, Person, etc. If I were to include a class diagram, most of the methods/attributes would have existed in AddressBook Level 4. Would you still recommend including a class diagram?

@yao-feng Having a class diagram would help the markers gain a better understanding of your UML Modelling skills. Also, while your implementation _may_ involve classes already existing in AB4, does your feature _create_ more functionality by being associated to those classes? 

Something to think about: Do you see a class diagram for the ModelManager or Person class anywhere in the developer guide? Assuming I were a new developer on your team with **no background** on what is AB4, would I know how your feature makes associations between those classes just by reading your documentation?

From the website: https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/project-deliverables.html

> Contributions to the Developer Guide: Reproduce the parts in the Developer Guide that you wrote. Ensure there is enough content to evaluate your technical documentation skills and UML modelling skills. You can include descriptions of your design/implementations, possible alternatives, pros and cons of alternatives, etc.

AceCentury commented:
Added v1.3 reviewed label as I forgot to add it the last time I reviewed >_<

yaofeng-wang commented:
@AceCentury Got it. Thanks for the review :thumbsup:

adjscent committed:
Merge pull request #183 from adjscent/master

increase test coverage

adjscent committed:
more sensible sample persons

kengjichow committed:
Merge pull request #186 from theJrLinguist/master

edit user guide, fix addEvent bug, add sample data

adjscent committed:
Merge branch 'master' into master

adjscent committed:
Merge pull request #187 from adjscent/master

more sensible sample persons

adjscent committed:
Fix spellings in guide

adjscent committed:
fix sampledatautil format

yaofeng-wang committed:
Refactor EditUser command and DeleteUser command

adjscent committed:
add a default ppt for adeel so that the link works

adjscent committed:
change jar reference to eventorganiser.jar

agendazhang committed:
Add permission for AddFriendCommand

adjscent committed:
Decouple Schedule

adjscent committed:
one more test case for time

agendazhang committed:
Add permission for DeleteFriendCommand

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  change jar reference to eventorganiser.jar
  add a default ppt for adeel so that the link works

adjscent committed:
Merge pull request #188 from adjscent/master

Break Schedule into smaller parts

agendazhang committed:
Fix style

agendazhang committed:
Fix DeleteUserCommand (now it cannot accept any parameters)

agendazhang committed:
Fix style

agendazhang committed:
Edit SuggestFriendsByInterests command

agendazhang committed:
Fix style

agendazhang committed:
Add new ListFriendsCommand

agendazhang committed:
Add tests for ListFriendsCommand

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  one more test case for time
  Decouple Schedule

yaofeng-wang committed:
Refactor findUser command

Commit:
edit TimePoll

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

Commit:
add overview to user guide

adjscent committed:
Delete RepoSense.jar

We do not need this jar file.

Commit:
update developer guide:

Commit:
fix date parse

kengjichow committed:
Merge pull request #191 from theJrLinguist/master

Update guides

agendazhang committed:
Fix style

adjscent committed:
Fix diagrams and remove reposense.jar

adjscent committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

adjscent committed:
fix user guide css

adjscent committed:
fix developer guide link

Commit:
fix PPP

kengjichow committed:
Merge pull request #192 from theJrLinguist/master

fix PPP

yaofeng-wang committed:
Resolve Issue #190

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

adjscent committed:
Fix empty timetable download.

adjscent committed:
Merge remote-tracking branch 'upstream/master'

adjscent committed:
Fix developer guide and user guide for timetable issues

adjscent committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

adjscent committed:
Increase user guide toc levels

adjscent committed:
Fix developer guide toc levels

agendazhang committed:
Fix style

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  Fix developer guide toc levels
  Increase user guide toc levels
  Fix developer guide and user guide for timetable issues
  fix PPP
  fix developer guide link
  fix user guide css
  Fix diagrams and remove reposense.jar
  fix date parse
  update developer guide:
  add overview to user guide
  edit TimePoll

adjscent committed:
Fix empty timetable download.

Commit:
edit PPP

agendazhang committed:
Update user guide and personal portfolio

adjscent committed:
eof issues

agendazhang committed:
Fix style

agendazhang committed:
Fix style

agendazhang committed:
Merge pull request #189 from agendazhang/master

Create permission for friend commands

adjscent committed:
Update DeveloperGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Merge branch 'master' into master

Commit:
fix FindEvent

kengjichow committed:
Merge branch 'master' into master

adjscent committed:
Merge pull request #193 from adjscent/master

Allow empty timetable download instead of crashing

Commit:
add EOF line config.json

Commit:
Merge branch 'master' of https://github.com/theJrLinguist/main

kengjichow committed:
Merge pull request #197 from theJrLinguist/master

Edit PPP, fix FindEvent

yaofeng-wang committed:
Resolve Issue #198

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

# Conflicts:
#	src/main/java/seedu/address/logic/commands/personcommands/DeleteUserCommand.java
#	src/main/java/seedu/address/logic/commands/personcommands/EditUserCommand.java

Commit:
edit PPP

Commit:
edit PPP

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

Commit:
edit UserGuide

agendazhang committed:
Fix EditUserCommand interest bug

adjscent committed:
Maxschedule test case

yaofeng-wang committed:
Resolve Issue #202

yaofeng-wang committed:
Improve test coverage for editUser command

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main into editUser

yaofeng-wang committed:
Update error message

adjscent committed:
Merge pull request #201 from adjscent/master

Maxschedule test case

yaofeng-wang committed:
Merge pull request #199 from yao-feng/master

Resolve Issue #190 and Issue #198

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main into editUser

yaofeng-wang committed:
Merge pull request #203 from yao-feng/editUser

editUser command

yaofeng-wang committed:
Increase test coverage for deleteUser command

yaofeng-wang committed:
Merge pull request #204 from yao-feng/master

Increase test coverage for deleteUser command

yaofeng-wang committed:
update UserGuide.adoc

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  Increase test coverage for deleteUser command
  Update error message
  Improve test coverage for editUser command
  Resolve Issue #202
  Maxschedule test case
  edit UserGuide
  edit PPP
  edit PPP
  Resolve Issue #198
  add EOF line config.json
  fix FindEvent
  edit PPP
  Resolve Issue #190
  Refactor findUser command
  Refactor EditUser command and DeleteUser command

Commit:
refactor tests and ModelManager

agendazhang committed:
Edit user guide

Commit:
fix editEvent

Commit:
add EOF line

kengjichow committed:
Merge pull request #205 from theJrLinguist/master

Minor edits, refactoring

adjscent committed:
Update README.adoc

agendazhang committed:
Fix tests

agendazhang committed:
Merge pull request #200 from agendazhang/master

Fix bugs and edit guides

adjscent committed:
Update UserGuide to specify the user uniqueness

agendazhang committed:
Fix EditUserCommand bug

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserPrefs.java

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update developer guide on slot, time and day objects

adjscent committed:
Update developer guide with proper repourl

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
update user and developer guide

agendazhang committed:
Edit user and developer guide

Commit:
fix findEvent

adjscent committed:
Add asserts and logging calls in schedule and timetable

adjscent committed:
fix typo for finduser in user guide

adjscent committed:
whitespace issue

adjscent committed:
whitespace issue

adjscent committed:
whitespace issue

adjscent committed:
whitespace issue

adjscent committed:
whitespace

adjscent committed:
whitespace

adjscent committed:
Update DeveloperGuide.adoc

adjscent committed:
Update DeveloperGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Merge branch 'master' into master

adjscent committed:
fix travis

adjscent committed:
fix travis

adjscent committed:
merge

adjscent committed:
fix travis

adjscent committed:
fix travis

adjscent committed:
fix commit

adjscent committed:
fix commit

agendazhang committed:
Fix FindUserCommand

agendazhang committed:
Fix style

Commit:
edit screenshots

yaofeng-wang committed:
Update documentations

yaofeng-wang committed:
Refactor code

adjscent committed:
Add asserts and logging calls in schedule and timetable (#208)

* Add asserts and logging calls in schedule and timetable

* fix typo for finduser in user guide

* whitespace issue

* whitespace issue

* whitespace

* Update DeveloperGuide.adoc

* Update UserGuide.adoc

* fix travis

* fix travis

* fix commit

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

agendazhang committed:
Edit InterestSimilarPredicate for SuggestFriendsByInterests command so now the search is non case sensitive

adjscent committed:
Update README.adoc

agendazhang committed:
Revert SuggestFriendsByInterests command back to original (case sensitive search)

adjscent committed:
merge

adjscent committed:
Merge branch 'master' of https://github.com/adjscent/main

adjscent committed:
Merge branch 'master' of https://github.com/adjscent/main

adjscent committed:
Merge pull request #211 from adjscent/master

Resync to main repo

adjscent committed:
Update DeveloperGuide.adoc

agendazhang committed:
Edit developer guide and fix tests

Commit:
fix duplicate event bugs

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main: (21 commits)
  Update DeveloperGuide.adoc
  fix commit
  fix travis
  fix travis
  Update UserGuide.adoc
  Update DeveloperGuide.adoc
  whitespace
  whitespace issue
  whitespace issue
  Update README.adoc
  Add asserts and logging calls in schedule and timetable (#208)
  fix commit
  fix travis
  fix travis
  Update UserGuide.adoc
  Update DeveloperGuide.adoc
  whitespace
  whitespace issue
  whitespace issue
  fix typo for finduser in user guide
  ...

# Conflicts:
#	docs/UserGuide.adoc

Commit:
fix style

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

kengjichow committed:
Merge pull request #212 from theJrLinguist/master

Fix findEvent, editEvent bugs

yaofeng-wang committed:
Update documents

Commit:
edit guides

Commit:
add EOF line

yaofeng-wang committed:
Proofread DeveloperGuide.adoc

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

yaofeng-wang committed:
Resolve Travis Issues

yaofeng-wang committed:
Merge pull request #210 from yao-feng/master

Update documentations

Commit:
add asserts

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

agendazhang committed:
Edit developer guide

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  Resolve Travis Issues
  Update documents
  Update documents
  Proofread DeveloperGuide.adoc
  add EOF line
  edit guides
  Update documents
  Update documents
  Update documents
  fix style
  fix duplicate event bugs
  Refactor code
  Update documentations
  edit screenshots
  fix findEvent
  update UserGuide.adoc

# Conflicts:
#	docs/DeveloperGuide.adoc

adjscent committed:
Update MaxScheduleCommand.java

agendazhang committed:
Merge pull request #207 from agendazhang/master

Edit developer guide

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge pull request #213 from yao-feng/master

Update documents

Commit:
edit user guide

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

Commit:
fix findEvent bug

Commit:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

agendazhang committed:
Edit developer guide

adjscent committed:
Update UserGuide.adoc

kengjichow committed:
Merge pull request #214 from theJrLinguist/master

Edit guide, fix bugs

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

agendazhang committed:
Edit Messages

agendazhang committed:
Add some asserts

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

agendazhang committed:
Merge pull request #215 from agendazhang/master

Edit developer guide

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

# Conflicts:
#	src/main/java/seedu/address/model/person/UserContainsKeywordsPredicate.java

yaofeng-wang committed:
Merge pull request #216 from yao-feng/master

Update documentation

Commit:
fix joinEvent bug

kengjichow committed:
Merge pull request #217 from theJrLinguist/master

fix joinEvent bug

Commit:
fix clearUser

Commit:
add EOF line

adjscent committed:
Change Ui.png

kengjichow committed:
Merge pull request #218 from theJrLinguist/master

Fix ClearCommand

agendazhang committed:
Change Person equals to name and password, Edit friend attribute

agendazhang committed:
Fix tests

yaofeng-wang committed:
Update documentation

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

# Conflicts:
#	src/main/java/seedu/address/MainApp.java

agendazhang committed:
Edit equals method for EditPersonDescriptor

yaofeng-wang committed:
Merge pull request #220 from yao-feng/master

Final group review of UserGuide.adoc

agendazhang committed:
Merge pull request #219 from agendazhang/master

Change Person equals to name and password, Edit friend attribute

adjscent committed:
fix schedule and timetable diagrams

adjscent committed:
fix schedule

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

yaofeng-wang committed:
Merge pull request #222 from yao-feng/master

Group review of DeveloperGuide.adoc

agendazhang committed:
Edit PP

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  Update documents
  fix schedule
  fix schedule and timetable diagrams
  Update documentation

agendazhang committed:
Merge pull request #223 from agendazhang/master

Edit PP

yaofeng-wang committed:
Update documents

adjscent committed:
Update adjscent.adoc

yaofeng-wang committed:
Merge pull request #224 from yao-feng/master

Update documents

Commit:
edit PPP

Commit:
add EOF line

kengjichow committed:
Merge pull request #225 from theJrLinguist/master

Edit PPP

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge pull request #226 from yao-feng/master

Update documents

agendazhang committed:
Fix tracking bug in developer guide

agendazhang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

* 'master' of https://github.com/CS2103-AY1819S1-W10-3/main:
  Update documents
  add EOF line
  edit PPP
  Update adjscent.adoc
  Update documents

agendazhang committed:
Merge branch 'master' into master

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge pull request #228 from yao-feng/master

Update documents

agendazhang committed:
Merge remote-tracking branch 'origin2/master'

agendazhang committed:
Delete TimeTableScheduleDiagram.xml

agendazhang committed:
Delete MaxScheduleDiagram.xml

agendazhang committed:
Delete ScheduleDiagram.xml

agendazhang committed:
Update adjscent.adoc

agendazhang committed:
Merge pull request #227 from agendazhang/master

Edit Developer Guide

Commit:
edit PPP

kengjichow committed:
Update README.adoc

kengjichow committed:
Merge pull request #229 from theJrLinguist/master

edit PPP

adjscent committed:
Update UserGuide.adoc

adjscent committed:
Update UserGuide.adoc

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W10-3/main

yaofeng-wang committed:
Merge pull request #230 from yao-feng/master

Update documents

yaofeng-wang committed:
Update documents

yaofeng-wang committed:
Merge pull request #231 from yao-feng/master

Update documents

kengjichow committed:
Update AboutUs

kengjichow committed:
Update AboutUs

adjscent committed:
Fix guides

Commit:
edit dev guide

kengjichow committed:
Merge pull request #232 from theJrLinguist/master

edit dev guide

kengjichow committed:
Minor edit

adjscent committed:
Cdn hash failure fix

Cdn has failed its caching so we need to change hash check to allow the html to display properly.

adjscent committed:
cdn fix continued

adjscent committed:
Reposense fix for adjscent

kengjichow committed:
add @@author tag

kengjichow committed:
Add @@author tag

kengjichow committed:
add @@author tag

kengjichow committed:
add @@author tag

kengjichow committed:
add @@author tag
